ui/webui/src/components/app.jsx

```diff
@@ -55,6 +55,8 @@ export const Application = () => {
     const [storeInitilized, setStoreInitialized] = useState(false);
 
     useEffect(() => {
+        // Before unload ask the user for verification
+        window.onbeforeunload = function (e) { return "" };
```

Comment:
Just out of curiosity, can't you do something like this?
```
window.onbeforeunload = e => "";
```
While I personally like the keyword variant more, I guess consistency would be more valuable...

Reply:
Yes, it can be written like that too